webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/wcif/model/ScrambleSet.kt

```diff
@@ -5,7 +5,8 @@ import org.worldcubeassociation.tnoodle.server.webscrambles.wcif.model.extension
 import org.worldcubeassociation.tnoodle.server.webscrambles.wcif.provider.IndexingIdProvider
 
 @Serializable
-data class ScrambleSet(override val id: Int, val scrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, val extraScrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, override val extensions: List<Extension> = emptyList()) : ExtensionProvider(), IndexingIdProvider {
+data class ScrambleSet(override val id: Int, val scrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, val extraScrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, override val extensions: List<Extension> = emptyList()
+) : ExtensionProvider(), IndexingIdProvider {
```

Comment:
Also, please try to figure out why IntelliJ doesn't do the automated line breaking like outlined above (or alternatively discard this change from this particular commit, adding only one weeny line break seems silly given the length of the line)

Reply:
It seems there's an option for that in editorconfig https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties but it's under "limited support"